tests/_testrunner.livecodescript

```diff
@@ -92,6 +92,13 @@ on DoQuit
    if "errorDialog" is in the pendingMessages then
  	  send "DoQuit" to me in 0 millisecs
    else
+      local tError
```

Comment:
Should this check be after each test rather than at the end. Perhaps in `TestRunnerInvokeTestCommand`? That way we know which test was the problem.